### PR TITLE
Scripts: OB recovery

### PR DESCRIPTION
**Pull request by skotopes**
# What's new

- Scripts: OB recovery

# Verification 

- Damage OB(not sure how), recover with `./scripts/ob.py -d recover --interface=interface/stlink.cfg`

# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: scripts/flipper/utils/programmer_openocd.py [hunk 1]
```diff
@@ -292,3 +292,19 @@ def otp_write(self, address: int, file_path: str) -> OpenOCDProgrammerResult:
             if validation_result
             else OpenOCDProgrammerResult.ErrorValidation
         )
+
+    def option_bytes_recover(self) -> bool:
+        self.openocd.start()
+        self.openocd.send_tcl("reset halt")
+        self.openocd.send_tcl("mww 0x58004010 0x8000")
+        self.openocd.send_tcl("mww 0x58004008 0x45670123")
```
Comment (DrZlo13) [inline scripts/flipper/utils/programmer_openocd.py]: `stm32.flash_unlock
`

File: scripts/flipper/utils/programmer_openocd.py [hunk 2]
```diff
@@ -292,3 +292,19 @@ def otp_write(self, address: int, file_path: str) -> OpenOCDProgrammerResult:
...
+        self.openocd.send_tcl("mww 0x58004008 0xCDEF89AB")
+        self.openocd.send_tcl("mww 0x5800400c 0x08192A3B")
```
Comment (DrZlo13) [inline scripts/flipper/utils/programmer_openocd.py]: `stm32.option_bytes_unlock`

**Review by DrZlo13 — CHANGES_REQUESTED** [review]

File: scripts/flipper/utils/programmer_openocd.py (hunk 2 above)
Comment (skotopes) [reply scripts/flipper/utils/programmer_openocd.py]: magic sequence

File: scripts/flipper/utils/programmer_openocd.py (hunk 1 above)
Comment (skotopes) [reply scripts/flipper/utils/programmer_openocd.py]: magic sequence

**Review by DrZlo13 — DISMISSED** [review]

**Review by DrZlo13 — APPROVED** [review]

**Pull request merged**